package.json

```diff
@@ -25,12 +25,17 @@
     "url": "https://github.com/ncb000gt/node.bcrypt.js/issues"
   },
   "scripts": {
-    "test": "node-gyp configure build && nodeunit test"
+    "test": "npm install --build-from-source && nodeunit test",
+    "install": "node-pre-gyp install --fallback-to-build"
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.3.5"
+    "nan": "2.3.5",
+    "node-pre-gyp": "0.6.4"
   },
+  "bundledDependencies": [
+    "node-pre-gyp"
```

Comment:
Hmm, looks like it changed here https://github.com/mapbox/node-pre-gyp/commit/8f7a774980a6aee1f4d4d2c36d8a27e2d3841334#diff-04c6e90faac2675aa89e2176d2eec7d8

I will remove it and test.